qubesappmenus/receive.py

```diff
@@ -48,6 +48,11 @@
     help='Only regenerate appmenus entries, do not synchronize with system '
          'in template')
 
+parser.add_argument(
+    '--run-as-root', action='store',
```

Comment:
In other tools similar option is named `--force-root` (see for example `qubesdmin/tools/__init__.py`), please make it this way here too.